docs/source/devguide.md

```diff
@@ -259,54 +259,116 @@ Make sure your change
+  by the works on this branch
+* `<title>` is either the name of the issue or an abbreviated version of it
 
 ## Release Process
 
-### xcube
+### Release on GitHub
+
+This describes the release process for `xcube`. For a plugin release, 
+you need to adjust the paths accordingly. 
 
 * Check issues in progress, close any open issues that have been fixed.
+* Make sure that all unit tests pass and that test coverage is 100% 
+  (or as near to 100% as practicable).
 * In `xcube/version.py` remove the `.dev` suffix from version name.
-* Make sure `CHANGES.md` is complete. Remove the suffix ` (in development)` from the last version headline.
+* Adjust version in `Dockerfile`  accordingly.
+* Make sure `CHANGES.md` is complete. Remove the suffix ` (in development)` 
+  from the last version headline.
 * Push changes to either master or a new maintenance branch (see above).
 * Await results from Travis CI and ReadTheDocs builds. If broken, fix.
-* Goto [xcube/releases](https://github.com/dcs4cop/xcube/releases) and press button "Draft a new Release".
+* Go to [xcube/releases](https://github.com/dcs4cop/xcube/releases) 
+  and press button "Draft a new Release".
   - Tag version is: `v${version}` (with a "v" prefix)
-  - Release title is: `${version}` 
+  - Release title is: `${version}` (without a "v" prefix) 
   - Paste latest changes from `CHANGES.md` into field "Describe this release"
   - Press "Publish release" button
 * After the release on GitHub, rebase sources, if the branch was `master`, 
   create a new maintenance branch (see above)
-* In `xcube/version.py` increase version number and append a `.dev0` suffix to version name so 
-  that it is still PEP-440 compatible.
+* In `xcube/version.py` increase version number and append a `.dev0` suffix 
+  to the version name so that it is still PEP-440 compatible.
+* Adjust version in `Dockerfile` accordingly.
 * In `CHANGES.md` add a new version headline and attach ` (in development)` to it.
 * Push changes to either master or a new maintenance branch (see above).
 * Activate new doc version on ReadTheDocs. 
 
-Go through the same procedure for all xcube plugin packages dependent on this version of xcube.
+Go through the same procedure for all xcube plugin packages 
+dependent on this version of xcube.
+
+### Release on Conda-Forge
+
+These instructions are based on the documentation at 
+[conda-forge](https://conda-forge.org/docs/maintainer/updating_pkgs.html).
+
+Conda-forge packages are produced from a github feedstock repository belonging 
+to the conda-forge organization. A repository's feedstock is usually located at 
+`https://github.com/conda-forge/<repo-name>-feedstock`, e.g., 
+https://github.com/conda-forge/xcube-feedstock.
+The package is updated by 
+* forking the repository
+* creating a new branch for the changes
+* creating a pull request to merge this branch into conda-forge's feedstock repository. 
```

Comment:
The pull request is automatically done by the regro-cf-autotick-bot  on release. The steps of creating a PR is only necessary if you need to update the current version i.e. build >0.

Reply:
I don't think that this applies in our case. [The conda-forge docs say](https://conda-forge.org/docs/maintainer/updating_pkgs.html#pushing-to-regro-cf-autotick-bot-branch):

> When a new version of a package is released on PyPI/CRAN/.., we have a bot that automatically creates version updates for the feedstock.

But we don't release on PyPI and the bot doesn't seem to track releases that only happen on GitHub. At least, I didn't notice any automatically created branches for any for the recent xcube / xcube-* releases.

(There's also the problem that we need to manually update dependencies for the conda releases, but I guess that would work if a bot created the PR too -- we just add our own changes before merging the PR.)

Reply:
Oops, I was wrong: https://github.com/conda-forge/xcube-feedstock/pull/13 is indeed there for xcube 0.6.1. So evidently it does track GitHub releases after all.